python/lsst/ts/gis/csc.py

```diff
@@ -75,8 +75,13 @@ async def telemetry_loop(self):
         self.log.info("Starting telemetry loop.")
         while True:
             try:
-                await self.component.update_status()
-                await asyncio.sleep(self.telemetry_interval)
+                if self.connected:
+                    await self.component.update_status()
+                    await asyncio.sleep(self.telemetry_interval)
+                else:
+                    await self.fault(
+                        code=1, report="Unexpectedly disconnected from GIS."
```

Comment:
Instead of `1`, how about putting it to a constant or enum? I used the enum in the M2 Python code to give the extensibility in the future as:
https://github.com/lsst-ts/ts_m2/blob/develop/python/lsst/ts/m2/enum.py#L27-L30